QuizApp/App/Common/Coordinator.swift

```diff
@@ -1,26 +1,20 @@
 import UIKit
+import Factory
 
 final class Coordinator: CoordinatorProtocol {
 
-    private let navigationController: UINavigationController
-    private let serviceFactory: ServiceFactoryProtocol
-
-    init(navigationController: UINavigationController) {
-        self.navigationController = navigationController
-        self.serviceFactory = ServiceFactory()
-    }
+    @Injected(Container.navigationController) private var navigationController
+    @Injected(QuizContainer.quizUseCase) private var quizUseCase
+    @Injected(UserContainer.userUseCase) private var userUseCase
 
     func showLogin() {
-        let loginViewModel = LoginViewModel(loginUseCase: serviceFactory.loginUseCase, coordinator: self)
-        let loginViewController = LoginViewController(loginViewModel: loginViewModel)
+        let loginViewController = LoginViewController()
         navigationController.setViewControllers([loginViewController], animated: true)
     }
 
     func showHome() {
-        let quizViewModel = QuizListViewModel(quizUseCase: serviceFactory.quizUseCase, coordinator: self)
-        let quizViewController = QuizListViewController(quizViewModel: quizViewModel)
-        let userViewModel = UserViewModel(userUseCase: serviceFactory.userUseCase)
-        let userViewController = UserViewController(userViewModel: userViewModel)
+        let quizViewController = QuizListViewController()
+        let userViewController = UserViewController()
```

Comment:
Why are view controllers not resolved in Container?

Reply:
I've tried to resolve the view controllers in the Container, same as view models, but the app crashes if I do so.